### ISSUE TITLE
Deploy to GH Pages using deploy key instead of API token

### ISSUE THREAD

**Issue by willfurnass** [DiODeProject/MuMoT#243]
This approach should be more secure.  NB need a custom script for this at present as the Travis GitHub Pages deployment mechanism [doesn't currently support GH Deploy Keys](https://github.com/travis-ci/dpl/issues/694). 

[Example of a custom script in the first comment here](https://github.com/travis-ci/dpl/issues/694)

**Comment by jarmarshall** [comment]
Thanks @willfurnass - great to see GitHub pages working with hyperlinks following PR #230 - for info, where is the version pulled from that creates the identifier `MuMoT 0.1.0-dev documentation`?

So you're taking care of this issue, right? The process is a bit opaque to me...

**Comment by willfurnass** [comment]
@jarmarshall Yes, happy to address this issue.  Shouldn't block releasing 1.0. 

FYI the version info in the Sphinx docs is derived from the package version; from `docs/source/conf.py`: 

```python
# ...
# The short X.Y version.
import mumot
version = mumot.__version__
# The full version, including alpha/beta/rc tags.
release = mumot.__version__
# ...
```

**Comment by willfurnass** [comment]
Redundant and can be closed if happy with proposal in #293 and implementation in #295
